### ISSUE TITLE
Hard to read language selector

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla/addons#10805]
Steps to reproduce:
1. Load AMO-dev homepage https://addons-dev.allizom.org/en-US/firefox/
2. Go to page footer and observe language selector

Expected results:
The language selector is clear and readable.

Actual results:
The language is written using black on gray background with icon and drop-down arrow in white.

Notes/Issues:
Maybe we can use white background? Or white color font for language as in production.
Verified on AMO-dev FF55(Win 7). 
Screenshot for this issue:
![2017-09-22_0939](https://user-images.githubusercontent.com/15685960/30732196-087039d2-9f7a-11e7-9f42-fdac31330403.png)
